src/Uno.UI/UI/Xaml/ResourceHelper.cs

```diff
@@ -1,7 +1,9 @@
-﻿using Uno.UI;
+﻿#if __ANDROID__ || __IOS__ || __MACOS__
```

Comment:
I don't recall, why android/ios only?

Reply:
These are the platforms where `ResourcesService` is instantiated:

https://github.com/unoplatform/uno/blob/bebb99ed518c7a86ecafc51840b9c4dca36438ac/src/Uno.UI/UI/Xaml/NativeApplication.cs#L46

https://github.com/unoplatform/uno/blob/bebb99ed518c7a86ecafc51840b9c4dca36438ac/src/Uno.UI/UI/Xaml/Application.macOS.cs#L45

https://github.com/unoplatform/uno/blob/bebb99ed518c7a86ecafc51840b9c4dca36438ac/src/Uno.UI/UI/Xaml/Application.iOS.cs#L41

`ResourceHelper.ResourcesService` is not set on Skia or Wasm, and `ResourcesService` isn't instantiated for them too.

So this class does "nothing" for those platforms.

I'm actually thinking now that we should make this whole ResourceHelper internal

Reply:
It's indeed a legacy class that we can remove or make internal.

Reply:
I added it to the issue and will open a separate PR for it.